### PR TITLE
Server path prefix

### PR DESCRIPTION
**Pull request by connortechnology**
Add a PathPrefix option to the Server object so that each Server in a multi-server setup could have a different URL scheme. Defaults to /zm

### CODE REVIEW

**Comment by connortechnology** [comment]
I havn't used these changes in montage review yet.

**Comment by pliablepixels** [comment]
Tagging myself here as I need to incorporate these changes on zmN when this is merged
https://github.com/pliablepixels/zmNinja/issues/724

File: web/skins/classic/views/js/montagereview.js.php [hunk 1]
```diff
@@ -118,13 +118,13 @@
   echo "var maxScore=$maxScore;\n";  // used to skip frame load if we find no alarms.
 } // end if initialmodeislive
 
-echo "var Storage = [];\n";
+echo "\nvar Storage = [];\n";
```
Comment (knight-of-ni) [inline web/skins/classic/views/js/montagereview.js.php:121]: The \n at the beginning there looks a bit odd. Just checking this is by design.

Comment (connortechnology) [reply web/skins/classic/views/js/montagereview.js.php:121]: Mostly there for readability of resulting javascript. separates logical sections with a new line.

**Comment by knight-of-ni** [comment]
Preliminary testing is successful.  Merging to get these change out to a wider audience....

**Pull request merged**